tests/gold_tests/pluginTest/lua/lua_states_stats.test.py

```diff
@@ -67,36 +71,27 @@
 ps.Env = ts.Env
 ps.ReturnCode = 0
 ps.Streams.stdout.Content = Testers.ContainsExpression("proxy.config.plugin.lua.max_states: 4", "expected 4 states")
-tr.TimeOut = 5
 tr.StillRunningAfter = ts
 
 # 1 Test - Exercise lua script
 tr = Test.AddTestRun("Lua hello")
 ps = tr.Processes.Default  # alias
 ps.Command = curl_and_args + ' http://hello/hello'
-ps.TimeOut = 5
 ps.ReturnCode = 0
 ps.Streams.stderr.Content = Testers.ContainsExpression("Hello, World", "hello world content")
-tr.TimeOut = 5
 tr.StillRunningAfter = ts
 
 # 2 Test - Check for metrics
 tr = Test.AddTestRun("Check for metrics")
-tr.DelayStart = 15  # 5s lag on metrics to update
-tr.TimeOut = 5
 ps = tr.Processes.Default  # alias
-ps.Env = ts.Env
-ps.Command = "traffic_ctl metric match lua"
+ps.Command = "bash -c '" + Test.TestDirectory + "/metrics.sh " + Test.TestDirectory + "'"
```

Comment:
Don't run stuff in the test directory. Copy this to the sandbox and run it

Reply:
Why?

Reply:
Because one can't replay the sandbox

Reply:
The argument against is that we want to minimize regression runtime.  Reducing unnecessary copying may reduce runtime.

Reply:
This is not a time suck. it is about reproducibility

Reply:
The time suck to I should clarify is all the sleeps and a lack of a way to detect that the metrics are up-to-date

Reply:
Hmm but the sandbox always has symlinks outside the sanbox.  So it's only reproducible if you maintain dependencies outside the sandbox in any case.